src/sdi-refresh-monitor.c

```diff
@@ -255,7 +260,7 @@ static void process_inhibited_snaps(SdiRefreshMonitor *self,
                               NULL);
       } else {
         // If we have snap data, we can use "pretty names" and icons
-        const gchar *snap_name = snapd_snap_get_name(client_snap);
+        const gchar *snap_name2 = snapd_snap_get_name(client_snap);
```

Comment:
I suppose tiobe complained about shadowing the existing `snap_name` variable here? In any case, I'd use a more expressive name - it's not clear to me how `snap_name` and `snap_name2` are different, since you're obtaining `client_snap` from `snap_name` via `snapd_client_get_snap_sync`, and then use `client_snap` to get `snap_name2` using `snapd_snap_get_name` (doesn't this just return the name you already have?_

Reply:
You are right... I think that the name is the same, but I prefer to wait to have all the unitary tests in place before modifying this. Reverted to the original code.